flowrs-build-designer/rete/flowrs/flowrsNode.ts

```diff
@@ -32,19 +31,40 @@ export class FlowrsNode extends Classic.Node<
                 typeParameters: { [key: string]: string } | null,
                 allPossibleTypes: Map<string, TypeDefinition>
     ) {
+        // TODO: @mafried Ich sollte davon ausgehen, dass der name eindeutig sein muss oder?
         super(name);
 
         this.constructor_type = constructor_type;
         this.setNodeData(data);
         this.setTypeParameters(typeParameters);
         let constructorDescription = this.getConstructorDescription(typeDefinition.constructors);
-        this.calculateCompatibleTypes(constructorDescription, allPossibleTypes);
+        // TODO @mafried bei Json constructorDescription füg ich ein TextInputField hinzu --> richtig ?
```

Comment:
Ja, wobei Nodes eher selten FromJson Konstruktoren haben (bisher, glaube ich, keiner).

Reply:
@mafried ich meine schon den Json und nicht den FromJson Constructor

Reply:
Den hat nämlich ValueNode z.b.

Reply:
ValueNode hat nur einen Konstruktor "New" vom Typ "NewWithArbitraryArgs", dieser Konstruktor hat zwei Argumente, wobei das erste Argument "value" heißt und einen Konstruktor mit Namen "Json" haben soll.

Reply:
Und wegen diesem Json Constructor füge ich ein Input Feld hinzu zum Node 
Das wäre dann richtig oder ?

Reply:
Ja, genau. Wir können das gerne auch noch jetzt gleich besprechen. Sie können unserem Zoom-Meeting jetzt schon beitreten, wenn Sie noch Fragen haben. Wir könnten dann auch das Problem mit dem fehlenden Trait-Support besprechen.

Reply:
Moment ich bereite alles vor und bin in 5 min da